backend/src/org/akvo/dash/transformation.clj

```diff
@@ -1,225 +1,157 @@
 (ns org.akvo.dash.transformation
```

Comment:
Could we try to normalize the terminology a bit regarding transformations?

Right now we're using at least three different words for the same thing: `op-spec`, `operation` and `transformation`. Personally I like `transformation` the most so I would suggest 

* `transformation`: a map with keys `:op`, `:args`, `:onError``
* `transformation-log`: A vector of transformations